CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ that users understand how the changes affect the new version.
 -->
 version 3.1.0
```

Comment:
changes should be added to a develop version

Reply:
Good catch, thank you. I have moved all changes that are not mentioned in release 3.1.0.